env/app_vars.sh

```diff
@@ -0,0 +1,112 @@
+echo "ALLOWED_HOSTS=["*"]" >>.env
+echo "AWS_STORAGE_BUCKET_NAME=ol-mitlearn-app-storage-qa" >>.env
+echo "CORS_ALLOWED_ORIGIN_REGEXES=['^.+ocw-next.netlify.app$']" >>.env
+echo "CSAIL_BASE_URL=https://cap.csail.mit.edu/" >>.env
+echo "CSRF_COOKIE_DOMAIN=.rc.learn.mit.edu" >>.env
+echo "EDX_API_ACCESS_TOKEN_URL=https://api.edx.org/oauth2/v1/access_token" >>.env
+echo "EDX_API_URL=https://api.edx.org/catalog/v1/catalogs/10/courses" >>.env
+echo "EDX_PROGRAMS_API_URL=https://discovery.edx.org/api/v1/programs/" >>.env
+echo "MICROMASTERS_CATALOG_API_URL=https://micromasters.mit.edu/api/v0/catalog/" >>.env
+echo "MICROMASTERS_CMS_API_URL=https://micromasters.mit.edu/api/v0/wagtail/" >>.env
+echo "MITOL_ADMIN_EMAIL=cuddle-bunnies@mit.edu" >>.env
+echo "MITOL_AXIOS_BASE_PATH=https://rc.learn.mit.edu" >>.env
+echo "MITOL_DEFAULT_SITE_KEY=micromasters" >>.env
+echo "MITOL_EMAIL_PORT=587" >>.env
+echo "MITOL_EMAIL_TLS=True" >>.env
+echo "MITOL_ENVIRONMENT=local" >>.env
+echo "MITOL_FROM_EMAIL=MITOpen <mitopen-support@mit.edu>" >>.env
+echo "MITOL_FRONTPAGE_DIGEST_MAX_POSTS=10" >>.env
+echo "MITOL_USE_S3=True" >>.env
+echo "MITOL_NOTIFICATION_EMAIL_BACKEND=anymail.backends.mailgun.EmailBackend" >>.env
+echo "MITPE_BASE_URL=https://professional.mit.edu/" >>.env
+echo "MITX_ONLINE_BASE_URL=https://mitxonline.mit.edu/" >>.env
+echo "MITX_ONLINE_COURSES_API_URL=https://mitxonline.mit.edu/api/v2/courses/" >>.env
+echo "MITX_ONLINE_LEARNING_COURSE_BUCKET_NAME=mitx-etl-mitxonline-production" >>.env
+echo "MITX_ONLINE_PROGRAMS_API_URL=https://mitxonline.mit.edu/api/v2/programs/" >>.env
+echo "NEW_RELIC_LOG=stdout" >>.env
+echo "NODE_MODULES_CACHE=False" >>.env
+echo "OCW_BASE_URL=https://ocw.mit.edu/" >>.env
+echo "OCW_CONTENT_BUCKET_NAME=ocw-content-storage" >>.env
+echo "OCW_UPLOAD_IMAGE_ONLY=True" >>.env
+echo "OCW_LIVE_BUCKET=ocw-content-live-production" >>.env
+echo "OLL_ALT_URL=https://openlearninglibrary.mit.edu/courses/" >>.env
+echo "OLL_API_ACCESS_TOKEN_URL=https://openlearninglibrary.mit.edu/oauth2/access_token/" >>.env
+echo "OLL_API_URL=https://discovery.openlearninglibrary.mit.edu/api/v1/catalogs/1/courses/" >>.env
+echo "OLL_BASE_URL=https://openlearninglibrary.mit.edu/course/" >>.env
+echo "OLL_LEARNING_COURSE_BUCKET_NAME=ol-data-lake-landing-zone-production" >>.env
+echo "OLL_LEARNING_COURSE_BUCKET_PREFIX=open-learning-library/courses/" >>.env
+echo "OPENSEARCH_DEFAULT_TIMEOUT=30" >>.env
+echo "OPENSEARCH_INDEXING_CHUNK_SIZE=75" >>.env
+echo "QDRANT_COLLECTION_NAME=mitlearn-qa" >>.env
+echo "PROLEARN_CATALOG_API_URL=https://prolearn.mit.edu/graphql" >>.env
+echo "SEE_API_URL=https://mit-unified-portal-prod-78eeds.43d8q2.usa-e2.cloudhub.io/api/" >>.env
+echo "SEE_API_ACCESS_TOKEN_URL=https://mit-unified-portal-prod-78eeds.43d8q2.usa-e2.cloudhub.io/oauth/token" >>.env
+echo "SECURE_CROSS_ORIGIN_OPENER_POLICY=None" >>.env
+echo "SEE_BASE_URL=https://executive.mit.edu/" >>.env
+echo "SOCIAL_AUTH_OL_OIDC_KEY=ol-mitlearn-client" >>.env
```

Comment:
This client will need to be set up with the local hostname redirect urls configured, are we ok with the same client for RC being setup like that? There's also some cases where keycloak will redirect to the client's main url, which will end the user up on RC instead of their local setup.

Reply:
I think we have two use cases around this:
- Dev wanting to go through the flow and login to QA/RC which this would cover
- Dev wanting to login through Keycloak to their own instance of learn. I think in this case, I'll create a separate client. The trick is the redirect as that's dynamic, however, we can setup a wildcard (not ideal) however between requiring the `client_secret` and enabling PKCE, we should be on the safer side.